index-handlers/src/main/java/no/sikt/nva/nvi/index/SearchNviCandidatesHandler.java

```diff
@@ -88,11 +88,13 @@ private CandidateSearchParameters getCandidateSearchParameters(RequestInfo reque
                                .orElse(List.of(topLevelOrg));
         var username = requestInfo.getUserName();
         var year = extractQueryParamPublicationDateOrDefault(requestInfo);
+        var category = Optional.ofNullable(extractQueryParamCategoryOrDefault(requestInfo))
+            .orElse(null);
```

Comment:
why is this ofNullable? extractQueryParamCategoryOrDefault returns either the category as a string, OR an empty string "".
It cant return null